templates/CRM/Contribute/Form/Selector.tpl

```diff
@@ -76,7 +73,7 @@
               {$row.$columnName|crmDate}
             </td>
           {else}
-          <td class="crm-{$columnName} crm-{$columnName}_{if $row.columnName}{$row.columnName}{/if}">
```

Comment:
This was added in https://github.com/civicrm/civicrm-core/commit/4fb5fcf3b17af6c9f5bf49ecc69902c5b0b78c24#diff-c7610d9326caa00eb91b61620c85f642bcf02174777b006d10f711fd76e87c4aR93 - however it seems to have been an error. Probably it should have been `{$row.columnName}` - but given it appears to be long-term broken I'd rather just remove

Reply:
You mean it should have been `{$row.$columnName}` using the variable instead of string literal as the array key?

Reply:
@colemanw yeah that's what I think - I can't find any evidence of the columnName key ever being set